api-catalog-services/src/main/java/org/zowe/apiml/apicatalog/standalone/StandaloneLoaderService.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+package org.zowe.apiml.apicatalog.standalone;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.netflix.appinfo.InstanceInfo;
+import com.netflix.discovery.shared.Applications;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.FilenameUtils;
+import org.apache.commons.io.IOUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Service;
+import org.zowe.apiml.apicatalog.instance.InstanceInitializeService;
+import org.zowe.apiml.apicatalog.services.cached.CachedApiDocService;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.util.List;
+
+@Slf4j
+@Service
+@ConditionalOnProperty(
+    value = "apiml.catalog.standalone.enabled",
+    havingValue = "true")
+@RequiredArgsConstructor
+public class StandaloneLoaderService {
+
+    @Value("${apiml.catalog.standalone.servicesDirectory:services}")
+    private String servicesDirectory;
+
+    private final ObjectMapper objectMapper;
+    private final InstanceInitializeService instanceInitializeService;
+    private final CachedApiDocService cachedApiDocService;
+    private final StandaloneAPIDocRetrievalService standaloneAPIDocRetrievalService;
+
+    public void initializeCache() {
```

Comment:
What is this method called? Only in `StandaloneInitializer`? What about joining those classes and changing visibility?